### PR TITLE
WIP Continuous integration

### PR DESCRIPTION
**Pull request by PenneyGadget**
This integrates Travis CI for continuous integration. It's unclear to me right now how exactly this will function on the master branch. Currently the Travis CI account is linked to me as I set up the account with my email. The docs say that the test coverage will appear in the pull requests, so we will see if that happens when I submit this. 
Also, one of of the acceptance criteria for this issue is to update the README with a link to where the test coverage can be viewed. I'm not sure of where that's happening yet / how it will function, so I will update the README and Basecamp with information as I know more - probably after this gets merged into master and what this adds becomes more clear.

### CODE REVIEW

**Comment by rrgayhart** [comment]
Looks good! 

It might be difficult with all of the changes on the source of this project moving around - but you'll need to squash down the commits before we merge this PR. Here's some instructions on how to do that: https://robots.thoughtbot.com/git-interactive-rebase-squash-amend-rewriting-history

Since the changes in this PR are small - I would first try squashing and if you run into problems with it - go back to master, create a new branch, redo the work there in one commit and push that up and delete this branch.

File: spec/controllers/api/v1/jobs_controller_spec.rb [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 RSpec.describe Api::V1::JobsController, type: :controller do
   describe "GET #index" do
-    let(:response_body) { json_respone = JSON.parse(response.body) }
+    let(:response_body) { JSON.parse(response.body) }
```
Comment (rrgayhart) [inline spec/controllers/api/v1/jobs_controller_spec.rb]: Good catch!

File: config/database.yml [hunk 2]
```diff
@@ -58,7 +58,7 @@ development:
 # Do not set this db to the same as development or production.
 test:
   <<: *default
-  database: lookingfor_test
+  database: travis_ci_test
```
Comment (rrgayhart) [inline config/database.yml]: Does this have any effect on your ability to run your tests locally?

**Comment by PenneyGadget** [comment]
This PR is from before the repo got deleted and brought back and is no longer relevant. There is a new Travis PR.

**Pull request closed without merging**